Explorer/Assets/DCL/PluginSystem/World/TweenPlugin.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Arch.SystemGroups;
+using Cysharp.Threading.Tasks;
+using DCL.ECSComponents;
+using DCL.Optimization.Pools;
+using DCL.PluginSystem.World.Dependencies;
+using DCL.SDKComponents.Tween.Components;
+using DCL.SDKComponents.Tween.Systems;
+using ECS.LifeCycle;
+using ECS.LifeCycle.Systems;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace DCL.PluginSystem.World
+{
+    public class TweenPlugin : IDCLWorldPlugin
```

Comment:
Use `IDCLWorldPluginWithoutSettings` instead to not re-implement empty methods

Reply:
Nice :) didnt know it existed!